### ISSUE TITLE
Issue in Payment (Receipt) Entry

### ISSUE THREAD

**Issue by nitin-anantpranali** [frappe/education#172]
While Education Module is anabled, in Payment Doctype Selection od Fees is not allowd. It comes automatically when we create Payment Entry from Fees DocTYpe.
![IMAGE 2023-12-02 11:16:35](https://github.com/frappe/education/assets/116695322/4058d386-e3c3-4453-9064-1599b9235658)

**Comment by RitvikSardana** [comment]
When you try to create a Payment Entry from "Fees" Doctype for that student, then you will be able to create a Payment Entry without any issue.

**Comment by nitin-anantpranali** [comment]
Yes, agreed. By practically to create entries for multiple students, it's a very lengthy flow.  

I mean the user have to go to the "Fees", Search the student from a huge list, Open the Fees DocType, Then go to Create Payment option.

**Comment by RitvikSardana** [comment]
so, what do you think should be an ideal way?
Bulk Fees Payment Tool?

Also we are thinking of converting Fees toSales Invoice

**Comment by nitin-anantpranali** [comment]
Bulk Payment is required for pending or backdated entries.

In case of a scenario where a dedicated person is allowed to accept the Fees having a huge line of students standing infant of him.

Converting to invoice would not be a correct way.

Think like a POS solution.

**Comment by RitvikSardana** [comment]
Converting to Invoice, can solve multiple issues Like

1. Advance Payment
2. Discount
3. Payment Terms
etc...

**Comment by nitin-anantpranali** [comment]
Hi,

I think, the current module is missing an important part, Fees Collection. Creating an another DocType will solve these purposes. 

1. Advance Payments (Advance Fees).
2. Discounts 
3. Payment Terms
4. Printing of Receipts
5. Searching Old receipts.
6. One Payment Settlement Entry for a given period.

How should this DocType behave ...
1. New DocType Fees Collection
2. Further Link this DocType to Payment with a option as Receipt wise Settlement or Day wise Settlement in Accounts (Like POS entries)
3. Will be having separate roles where Non accounting Department (Admission Department) will do their work separately.
